Resources/Prototypes/Entities/Structures/Machines/lathe.yml

```diff
@@ -753,14 +755,6 @@
       - WeaponLaserCannon
       - WeaponLaserCarbine
       - WeaponXrayCannon
-      - PowerCageSmall
-      - PowerCageMedium
-      - PowerCageHigh
-      - ShuttleGunSvalinnMachineGunCircuitboard
-      - ShuttleGunPerforatorCircuitboard
-      - ShuttleGunFriendshipCircuitboard
-      - ShuttleGunDusterCircuitboard
```

Comment:
Did you intend to remove all of these?

Reply:
"I also deleted some recipes in the secfab that were duplicated for some reason and moved PortableRechargers up to where they should be alphabetically"